components/contact/ContactSection.js

```diff
@@ -15,8 +21,7 @@ export default function ContactSection({ title, intro, id, details }) {
           ContactSectionRow({ ...x, index, detail: x.items[0].content })
         )}
       </div>
-      <p></p>
-      <div className="grid grid-cols-4 gap-4 border-t-2 pb-6" />
+      <div className="grid grid-cols-4 gap-4 mt-4 border-t-2 pb-6" />
```

Comment:
What is the grid for here? Is there content supposed to be inserted at the bottom?

Reply:
A grey line is supposed to be inserted here. I've removed the grid from the styles.